services/kaze/controllers/application.go

```diff
@@ -228,29 +230,36 @@ func TransferApplicationOwnership(c *gin.Context) {
 
 // FetchMetrics retrieves the metrics of an application's container
 func FetchMetrics(c *gin.Context) {
+	c.Writer.Header().Set("Content-Type", "text/event-stream")
 	appName := c.Param("app")
-	filter := utils.QueryToFilter(c.Request.URL.Query())
-	var timeSpan int64
-
-	for unit, converter := range timeConversionMap {
-		if val, ok := filter[unit].(string); ok {
-			timeVal, err := strconv.ParseInt(val, 10, 64)
-			if err != nil {
-				continue
-			}
-			timeSpan += timeVal * converter
-		}
+
+	metricsFetchInterval := c.Query("fetch_interval")
+	fetchInt, err := strconv.ParseInt(metricsFetchInterval, 10, 64)
```

Comment:
the line `metricsFetchInterval := c.Query("fetch_interval")` is unnecessary
You can just do `fetchInt, err := strconv.ParseInt(c.Query("fetch_interval"), 10, 64)`

Doing this might prevent an extra register being used unnecessarily in case the compiler's assembly optimization was not good enough

Same for the "fetch_counts" part